app/ui/src/scss/utils/_overrides.scss

```diff
@@ -15,33 +15,31 @@ body.cards-pf {
   margin-bottom: 10px;
 }
 
-.layout-pf.layout-pf-fixed .container-pf-nav-pf-vertical.collapsed-nav {
-  margin-left: 0;
+.pf-c-page {
+  display: block;
+  grid-template-columns: none;
+  grid-template-rows: none;
+  grid-template-areas: none;
 }
 
-.navbar {
-  a {
-    color: #ffffff;
-  }
+.pf-c-page__header {
+  grid-area: none;
```

Comment:
Same here, if the parent isn't `display: grid;`, I would expect this to be ignored